docs/getting-started/supported-languages-and-tools.md

```diff
@@ -1,17 +1,16 @@
 ---
-description: List of all the programming languages that Codacy supports and the corresponding tools that Codacy uses to analyze your source code.
+description: List of tools that Codacy uses to analyze over 40 supported languages and frameworks. Codacy provides static analysis for all programming languages and cloud infrastructure-as-code frameworks as well as code duplication, code complexity, and test coverage metrics for most programming languages.
 ---
 
 # Supported languages and tools
 
-For each supported programming language, Codacy generally provides:
+Codacy uses industry-leading tools to perform automatic static code analysis over 40 supported languages and frameworks:
 
--   Static Analysis
--   Code Duplication
--   Code Complexity
--   Test Coverage
+-   **For programming languages** Codacy provides Static Analysis as well as Code Duplication, Code Complexity, and Test Coverage metrics for key languages.
```

Comment:
I think that here it's safe to say that besides Static Analysis we also support more metrics **for key languages**? From my perspective, the most important ones are already covered with duplication, complexity, and coverage.